### PR TITLE
Change product pack usage to zip files

### PR DESCRIPTION
**Pull request by Savidude**

### CODE REVIEW

File: dev/group_vars/apim-analytics.yml [hunk 1]
```diff
@@ -16,16 +16,20 @@
...
+jdbc_driver: mysql-connector-java-5.1.47-bin.jar
+
+# JDK Distributions
+jdk_name: amazon-corretto-8.202.08.2-linux-x64
+deb_lib_dir: /usr/lib/wso2
+rpm_lib_dir: /usr/lib64/wso2
+
+# Installation directory information
+distribution_path: "{{ product_path }}/{{ product_name }}/{{ product_version }}"
+install_path: "{{distribution_path}}/{{product_name}}-{{product_version}}"
```
Comment (VimukthiPerera) [inline dev/group_vars/apim-analytics.yml]: Missing spaces in {{distribution_path}}. Should be {{ distribution_path }}

File: roles/apim-analytics-worker/tasks/main.yml [hunk 2]
```diff
@@ -14,64 +14,61 @@
...
-        dest: "{{ setup_path }}/{{ product_name }}/{{ product_version }}/conf/worker/deployment.yaml"
+    # WSO2 Distribution
+
+    - name: Create distribution path
+      file:
+        path: "{{ distribution_path }}"
+        state: directory
+        mode: 0755
+
+    - name: Unzip WSO2 Identity Server Package
```
Comment (VimukthiPerera) [inline roles/apim-analytics-worker/tasks/main.yml]: Should be API Manager

File: site.yml [hunk 3]
```diff
@@ -34,6 +34,12 @@
   roles:
     - apim
 
+- name: Apply API Management store configuration to an apim node
+  hosts:
+   - apim-is-as-km_1
+  roles:
+   - apim-is-as-km
```
Comment (VimukthiPerera) [inline site.yml]: Shall we leave the site.yml as it is? If someone needs the IS as Key Manager, they can add it here. Instructions are/should be given in the README.

File: roles/apim/tasks/main.yml [hunk 4]
```diff
@@ -14,99 +14,61 @@
...
+      file:
+        path: "{{ distribution_path }}"
+        state: directory
+        mode: 0755
 
-    - name: "Copy catalina-server.xml"
-      template:
-        src: "carbon-home/repository/conf/tomcat/catalina-server.xml.j2"
-        dest: "{{ setup_path }}/{{ product_name }}/{{ product_version }}/repository/conf/tomcat/catalina-server.xml"
+    - name: Unzip WSO2 Identity Server Package
```
Comment (VimukthiPerera) [inline roles/apim/tasks/main.yml]: API Manager package

File: roles/apim-tm/tasks/main.yml [hunk 5]
```diff
@@ -14,99 +14,61 @@
...
+      file:
+        path: "{{ distribution_path }}"
+        state: directory
+        mode: 0755
 
-    - name: "Copy catalina-server.xml"
-      template:
-        src: "carbon-home/repository/conf/tomcat/catalina-server.xml.j2"
-        dest: "{{ setup_path }}/{{ product_name }}/{{ product_version }}/repository/conf/tomcat/catalina-server.xml"
+    - name: Unzip WSO2 Identity Server Package
```
Comment (VimukthiPerera) [inline roles/apim-tm/tasks/main.yml]: API Manager package

Comment (Savidude) [reply roles/apim-tm/tasks/main.yml]: Fixed https://github.com/wso2/ansible-apim/pull/21/commits/cbf501d33b78acaf4e80694eb13ce8b2b43991e9

File: roles/apim/tasks/main.yml (hunk 4 above)
Comment (Savidude) [reply roles/apim/tasks/main.yml]: Fixed https://github.com/wso2/ansible-apim/pull/21/commits/cbf501d33b78acaf4e80694eb13ce8b2b43991e9

File: roles/apim-analytics-worker/tasks/main.yml (hunk 2 above)
Comment (Savidude) [reply roles/apim-analytics-worker/tasks/main.yml]: Fixed https://github.com/wso2/ansible-apim/pull/21/commits/cbf501d33b78acaf4e80694eb13ce8b2b43991e9

File: dev/group_vars/apim-analytics.yml (hunk 1 above)
Comment (Savidude) [reply dev/group_vars/apim-analytics.yml]: Fixed https://github.com/wso2/ansible-apim/pull/21/commits/cbf501d33b78acaf4e80694eb13ce8b2b43991e9

File: site.yml (hunk 3 above)
Comment (Savidude) [reply site.yml]: Fixed https://github.com/wso2/ansible-apim/pull/21/commits/25770b373ef66ef006d79d5c6e281b3bd54b0f65

**Pull request merged**